language-support/ts/codegen/tests/daml/daml.yaml

```diff
@@ -1,16 +0,0 @@
-# Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
-# SPDX-License-Identifier: Apache-2.0
```

Comment:
This file is actually not used, as the compilation is done with the `compile_daml` rule, which generates a `daml.yaml` file. It took me a while to figure out why my `build-options` didn't seem to take effect, so I thought I'd save other people future confusion by just taking it out.